pkg/driver/node.go

```diff
@@ -419,6 +474,28 @@ func parseVolumeId(volumeId string) (fsid, subpath, apid string, err error) {
 	return
 }
 
+func (p podAuth) setPodCredentials() (e error) {
+
+	response, err := k8sv1.CoreV1Interface.ServiceAccounts(p.k8sClient, p.nameSpace).Get(p.ctx, p.serviceAccount, metav1.GetOptions{})
```

Comment:
I don't think it's enormously important to pass the context through here, you could remove it and then it would make the type simpler, I could be wrong though!